pkg/database/accountingdb.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package database
+
+import dbx "storj.io/storj/pkg/database/dbx"
+
+// AccountingDB interface for AccountingDB database operations
+type AccountingDB interface {
```

Comment:
The interfaces would be in the package that use it, not the other way around.

Reply:
But then we will dependency cycle e.g. database <-> statdb. Database will need info about statdb interface and statdb will need info about database dbx part.

Reply:
statdb shouldn't need to know anything about dbx.

Reply:
Ehh ok got it. Old java habits, I don't need to implement directly interfaces, I just need a set of methods.